src/main/java/org/fcrepo/migration/PicocliMigrator.java

```diff
@@ -86,9 +86,13 @@ static F3SourceTypes toType(final String v) {
     private File f3ExportedDir;
 
     @Option(names = {"--target-dir", "-a"}, required = true, order = 5,
-            description = "Directory where OCFL storage root and supporting state will be written")
+            description = "OCFL storage root directory (data/ocfl-root is created for migration-type FEDORA_OCFL)")
     private File targetDir;
 
+    @Option(names = {"--working-dir", "-i"}, order = 6,
```

Comment:
If `--working-dir` is required (which it seems to be by line 197 (below)
> notNull(workingDir, "workingDir must be provided!");

then you should just add the `required = true` to this Option the same as for `--target-dir` above, then you won't need the `notNull` later on

Reply:
updated to use current dir if working-dir isn't passed in